spec/omniauth/strategies/saml_spec.rb

```diff
@@ -10,6 +10,15 @@ def post_xml(xml=:example_response, opts = {})
   post "/auth/saml/callback", opts.merge({'SAMLResponse' => load_xml(xml)})
 end
 
+def post_logout(params = nil)
```

Comment:
How about making this the `subject` of a new context block that surrounds the three context blocks that use this method instead? I think it's easier to follow the specs if this definition is near the examples it supports. I'm not much of a fan of the existing `post_xml` method for that reason.